modules/run-consul/run-consul

```diff
@@ -173,6 +181,42 @@ function assert_is_installed {
   fi
 }
 
+function generate_rpc_encryption_config {
```

Comment:
There's no need to write the RPC or gossip encryption configs to a separate config file. In fact, it makes it a bit harder for users to override the defaults. I'd write it to the same config as the rest of the configs written by `run-consul`.

Reply:
I did it this way because I felt like this would reduce complexity by not creating additional `if`/`else` code paths in the same `generate_consul_config`, but I'm ok with doing it that way if that's your preference.